packages/desktop-electron/index.ts

```diff
@@ -14,6 +14,8 @@ import {
   UtilityProcess,
 } from 'electron';
 import isDev from 'electron-is-dev';
+// @ts-strict-ignore
```

Comment:
Node fetch doesn't have types on the version we use - we'll need to upgrade it to get the types
https://www.npmjs.com/package/node-fetch#typescript

We do this elsewhere - copying for now until we upgrade the package